src/outcome_spaces/transfer_operator/transfer_operator.jl

```diff
@@ -206,7 +153,7 @@ rectangular partition given by the `binning`.
 """
 function transferoperator(pts::AbstractStateSpaceSet{D, T},
         binning::Union{FixedRectangularBinning, RectangularBinning};
-        boundary_condition = :circular, 
+        boundary_condition = :none, 
```

Comment:
I don't expect it to be breaking. My idea here was to avoid introducing an extra transition that's not present in the symbolic time series (hence changed the default from `:circular` to `:none`). But this definitely need to be discussed, and tests need to be added.
Plus, if the symbolic time series is long enough, the differences in results are expected to be negligible